pom.xml

```diff
@@ -2,18 +2,20 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <artifactId>spark-connector</artifactId>
-    <version>1.2.4</version>
+    <version>1.2.5</version>
     <name>The Spark Exasol Connector</name>
     <description>A connector for Apache Spark to access Exasol</description>
     <url>https://github.com/exasol/spark-connector/</url>
     <parent>
         <artifactId>spark-connector-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>1.2.4</version>
+        <version>1.2.5</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
     <properties>
         <log4j.version>2.20.0</log4j.version>
+        <sonar.sources>.</sonar.sources>
```

Comment:
I don't see those two new properties being used in the file?
What is their purpose here?

Maybe a comment would help.

Reply:
Yes, it is mainly for Sonar, since it doesn't find source files `src/scala` and coverage is not reported. I am going to add comment.

Reply:
Great, thanks.